gui/system/forms.py

```diff
@@ -1103,22 +1077,23 @@ def done(self, request, events):
                 protocol = 'http'
             else:
                 protocol = self.instance.stg_guiprotocol
+
             newurl = "%s://%s" % (
                 protocol,
                 address
             )
+
             if self.instance.stg_guiport and protocol == 'http':
                 newurl += ":" + str(self.instance.stg_guiport)
             elif self.instance.stg_guihttpsport and protocol == 'https':
                 newurl += ":" + str(self.instance.stg_guihttpsport)
-            notifier().start_ssl("nginx")
-            if self.instance._original_stg_guiprotocol != self.instance.stg_guiprotocol:
+
+            if self.original_instance['stg_guiprotocol'] != self.instance.stg_guiprotocol:
                 events.append("evilrestartHttpd('%s')" % newurl)
             else:
                 events.append("restartHttpd('%s')" % newurl)
-        if self.instance._original_stg_timezone != self.instance.stg_timezone:
-            os.environ['TZ'] = self.instance.stg_timezone
-            time.tzset()
```

Comment:
Why were these two lines removed?

Reply:
They have been removed because when we restart the timeservices, the tz variable is being set there  configured with the new time zone selected by the user. So they are unnecessary in the done method

Reply:
@sonicaj these are to change the timezone of the current django process. What am I missing?